pkg/validate/container.go

```diff
@@ -109,6 +112,40 @@ var _ = framework.KubeDescribe("Container", func() {
 			testExecSync(rc, containerID)
 		})
 	})
+
+	Context("runtime should support adding volume and device", func() {
+		var podID string
+		var podConfig *runtimeapi.PodSandboxConfig
+
+		BeforeEach(func() {
+			podID, podConfig = createPodSandboxForContainer(rc)
+			pullImageList(ic, []string{defaultContainerImage})
+		})
+
+		AfterEach(func() {
+			By("stop PodSandbox")
+			rc.StopPodSandbox(podID)
+			By("delete PodSandbox")
+			rc.RemovePodSandbox(podID)
+		})
+
+		It("runtime should support starting container with volume", func() {
```

Comment:
Add `[Conformance]` in every conformance test case. We will need this when we have benchmark.
`runtime should support starting container with volume [Conformance]`